.craft.yml

```diff
@@ -0,0 +1,13 @@
+minVersion: 0.23.1
+github:
+  owner: getsentry
+  repo: arroyo
+changelog: CHANGES.rst
+changelogPolicy: auto
+statusProvider:
+  name: github
```

Comment:
I think we may need to add a context for relying on github status.
https://github.com/getsentry/cdc/pull/38
and the fix for that https://github.com/getsentry/cdc/pull/39

Reply:
Removed statusProvider; I think we can just rely the defaults here like our SDKs do